cmd/webhook/app/webhook.go

```diff
@@ -101,12 +102,16 @@ func Run(ctx context.Context, opts *options.Options) error {
 		Logger: klog.Background(),
 		Scheme: gclient.NewSchema(),
 		WebhookServer: webhook.NewServer(webhook.Options{
-			Host:          opts.BindAddress,
-			Port:          opts.SecurePort,
-			CertDir:       opts.CertDir,
-			CertName:      opts.CertName,
-			KeyName:       opts.KeyName,
-			TLSMinVersion: opts.TLSMinVersion,
+			Host:     opts.BindAddress,
+			Port:     opts.SecurePort,
+			CertDir:  opts.CertDir,
+			CertName: opts.CertName,
+			KeyName:  opts.KeyName,
+			TLSOpts: []func(config *tls.Config){
+				func(config *tls.Config) {
+					config.MinVersion = uint16(opts.TLSMinVersion)
```

Comment:
It probably not work, as the acceptable value for the tls.MinVersion are:
```
	VersionTLS10 = 0x0301
	VersionTLS11 = 0x0302
	VersionTLS12 = 0x0303
	VersionTLS13 = 0x0304
```
See https://github.com/golang/go/blob/555af99bcc6b7515ebad5a3c947f3e4ab147fd29/src/crypto/tls/common.go#L29-L32.

Reply:
![image](https://github.com/karmada-io/karmada/assets/89568107/b36e120c-0799-400c-b445-4ad7f24b9cea)

It is recommended to add a parameter conversion method  @RainbowMango @liangyuanpeng

Reply:
+1 
on this approach.

Reply:
agree